### PR TITLE
Dlg spells

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/instat.vbproj [hunk 1]
```diff
@@ -2698,6 +2698,9 @@
     <EmbeddedResource Include="dlgDataFrameMetadata.sw-KE.resx">
       <DependentUpon>dlgDataFrameMetadata.vb</DependentUpon>
     </EmbeddedResource>
+    <EmbeddedResource Include="dlgSpells.resx">
+      <DependentUpon>dlgSpells.vb</DependentUpon>
+    </EmbeddedResource>
```
Comment (lilyclements) [inline instat/instat.vbproj]: So this automatically was added. I tried to undo it and that resulted in a loss of all my changes I made up to that point. Hopefully this line won't cause a conflict?

Comment (lilyclements) [inline instat/instat.vbproj]: So this automatically was added. I tried to undo it and that resulted in a loss of all my changes I made up to that point. Hopefully this line won't cause a conflict?

**Comment by lilyclements** [comment]
@rdstern perhaps I should screen share with you and run by how this is working? You can let me know if in the current state if it is okay. There is an issue if you do things in a certain order

File: instat/instat.vbproj (hunk 1 above)
Comment (dannyparsons) [inline instat/instat.vbproj]: As long as the resx file actually exists in the main branch already then this will be fine.

**Comment by dannyparsons** [comment]
@lilyclements How about briefly explaining to someone what's working so that someone can do a quick review and it gets merged?

**Comment by lilyclements** [comment]
Yep sure, so, this dialog is for defining dry spells. You need to do things to the date ranges before changing the interval for dry spell boxes.
@africanmathsinitiative/developers can someone review this please?

**Comment by lilyclements** [comment]
This needs to be in tomorrow's version, can someone review this?

**Comment by lilyclements** [comment]
@Lunalo I believe you reviewed this one earlier too. Can you let me know if there's anything I need to change? Otherwise, can you let Danny know it's ready for him

File: instat/dlgSpells.vb [hunk 2]
```diff
@@ -227,7 +227,7 @@ Public Class dlgSpells
         MaxValue()
     End Sub
 
-    Private Sub nudToAndFrom_ControlValueChanged(sender As Object, e As EventArgs) Handles nudTo.ValueChanged, nudFrom.ValueChanged
+    Private Sub nudToAndFrom_ControlValueChanged(sender As Object, e As EventArgs) Handles nudTo.TextChanged, nudFrom.TextChanged
```
Comment (Lunalo) [inline instat/dlgSpells.vb]: `nudToAndFrom_ControlValueChanged`
Naming problem

File: instat/dlgSpells.vb [hunk 3]
```diff
@@ -261,7 +261,7 @@ Public Class dlgSpells
         TestOKEnabled()
     End Sub
 
-    Private Sub conditionnuds_valuechanged(sender As Object, e As EventArgs) Handles NudConditionRight.ValueChanged, nudConditionLeft.ValueChanged
+    Private Sub conditionnuds_valuechanged(sender As Object, e As EventArgs) Handles NudConditionRight.TextChanged, nudConditionLeft.TextChanged
```
Comment (Lunalo) [inline instat/dlgSpells.vb]: `Sub conditionnuds_valuechanged`
Same as here

Comment (Lunalo) [inline instat/dlgSpells.vb]: Otherwise this looks good

File: instat/dlgSpells.vb (hunk 2 above)
Comment (lilyclements) [inline instat/dlgSpells.vb]: What would you suggest instead of this name?

**Comment by lilyclements** [comment]
@Lunalo I made the suggested changes.

**Comment by Lunalo** [comment]
@dannyparsons 

This is ready

**Pull request merged**